alf/utils/resnet50_block.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from tensorflow.keras import layers
+
+
+class BottleneckBlock(tf.keras.models.Model):
+    def __init__(self,
+                 kernel_size,
+                 filters,
+                 stage,
+                 block,
```

Comment:
No need to have `stage` and `block`, just `name` is enough.

Reply:
removed